controllers/operands/ssp_test.go

```diff
@@ -1265,7 +1265,130 @@ var _ = Describe("SSP Operands", func() {
 					commonImages = append(commonImages, image3)
 					commonImages = append(commonImages, image4)
 
-					Expect(ssp.Spec.CommonTemplates.DataImportCronTemplates).To(ContainElements(hcoDictSliceToSSP(commonImages)))
+					Expect(ssp.Spec.CommonTemplates.DataImportCronTemplates).To(ContainElements(hcoDictSliceToSSP(hco, commonImages)))
+				})
+
+				Context("heterogeneous cluster", func() {
+					BeforeEach(func() {
+						Expect(os.Mkdir(dir, os.ModePerm)).To(Succeed())
+						destFile := path.Join(dir, "dataImportCronTemplates.yaml")
+
+						Expect(
+							commontestutils.CopyFile(destFile, path.Join(testFilesLocation, "dataImportCronTemplatesMultiArch.yaml")),
+						).To(Succeed())
+
+						DeferCleanup(func() {
+							Expect(os.RemoveAll(dir)).To(Succeed())
+						})
+					})
+
+					It("should drop the ssp.kubevirt.io/dict.architectures annotation, when heterogeneous cluster is disabled (default)", func() {
```

Comment:
what does it mean "_when heterogeneous cluster is disabled_"?
i think the intention was:
"when EnableMultiArchCommonBootImageImport is disabled"
?

Reply:
re-wrote test names.